### ISSUE TITLE
Elevated CSF fumarate

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#6485]
**Preferred term label:**
An increased concentration of fumarate, an intermediate in the citric acid cycle, in the cebrebrospinal fluid.

Same for circulating

fumarate is a dicarboxcylic acid

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
https://pubmed.ncbi.nlm.nih.gov/31290619/
https://omim.org/entry/619060

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added
